pyproject.toml

```diff
@@ -7,6 +7,12 @@ license = { text = "GPL-2.0" }
 requires-python = ">=3.9"
 dependencies = []
 
+[project.urls]
+Homepage = "https://github.com/artefactual-labs/clamav-client"
+Documentation = "https://github.com/artefactual-labs/clamav-client/blob/main/README.md"
+Changes = "https://github.com/artefactual-labs/clamav-client/deployments"
+Code = "https://github.com/artefactual-labs/clamav-client"
```

Comment:
I can't find `Code` in https://packaging.python.org/en/latest/specifications/well-known-project-urls/, should we use `Repository` or `Source` instead?